### PR TITLE
refactor: update the font loading as in the document

### PR DESCRIPTION
**Pull request by ardasnturk**
- update font loading using `useFonts` as in the expo documentation
  - https://docs.expo.dev/develop/user-interface/fonts/#with-usefonts-hook

SVA-1400

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
-import * as Sentry from '@sentry/react-native';
 import AsyncStorage from '@react-native-async-storage/async-storage';
+import * as Sentry from '@sentry/react-native';
 import * as Font from 'expo-font';
```
Comment (donni106) [inline App.js:3]: `import { useFonts } from 'expo-font';` from the docu

File: App.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ SplashScreen.preventAutoHideAsync();
 
 const App = () => {
   const appState = useRef(AppState.currentState);
-  const [isFontLoaded, setIsFontLoaded] = useState(false);
+  const [isFontLoaded] = Font.useFonts(fontConfig);
```
Comment (donni106) [inline App.js:29]: wow, is this new since latest expo sdk updates?

File: App.js (hunk 1 above)
Comment (ardasnturk) [reply App.js]: https://github.com/smart-village-solutions/smart-village-app-app/commit/bb2bdc60ccd5f64a66494d86ad10a033f3cf4e4b

File: App.js (hunk 2 above)
Comment (ardasnturk) [reply App.js]: As far as I can see, this is a new hook that comes with SDK 50.

**Review by donni106 — APPROVED** [review]

**Pull request merged**